docker-compose.yml

```diff
@@ -24,35 +14,31 @@ services:
       - SPRING_PROFILES_ACTIVE=dev
       - APPLICATION_AUTHENTICATION_UI_ALLOWLIST=0.0.0.0/0
 
-  prison-register:
```

Comment:
Removing this config to run Prison Register locally as it no longer works - and there isn't much need to do it now.